From 644e45c7fa836d628de39badc3a168f3d7e9e9dc Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 25 Feb 2021 14:28:40 +0000
Subject: [PATCH] GCC: remove extra qualification from CrossThreadCopier

Extra qualification is invalid C++ and also not required here,
because it is already inside namespace WTF.
---
 .../blink/renderer/platform/graphics/resource_id_traits.h     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/blink/renderer/platform/graphics/resource_id_traits.h b/third_party/blink/renderer/platform/graphics/resource_id_traits.h
index b370d7c..565dfee 100644
--- a/third_party/blink/renderer/platform/graphics/resource_id_traits.h
+++ b/third_party/blink/renderer/platform/graphics/resource_id_traits.h
@@ -15,8 +15,8 @@
 namespace WTF {
 
 template <>
-struct WTF::CrossThreadCopier<viz::ResourceId>
-    : public WTF::CrossThreadCopierPassThrough<viz::ResourceId> {};
+struct CrossThreadCopier<viz::ResourceId>
+    : public CrossThreadCopierPassThrough<viz::ResourceId> {};
 
 template <>
 struct HashTraits<viz::ResourceId> : GenericHashTraits<viz::ResourceId> {
-- 
2.26.2

